samples/ControlCatalog.NetCore/Program.cs

```diff
@@ -96,6 +99,33 @@ void FormatMem(string metric, long bytes)
                 SilenceConsole();
                 return builder.StartLinuxDrm(args, scaling: GetScaling());
             }
+            else if (args.Contains("--vulkan"))
+            {
+                builder
+                    .With(new X11PlatformOptions
+                    {
+                        EnableMultiTouch = true,
+                        UseDBusMenu = true, 
+                        EnableIme = true, 
+                        UseGpu = false
+                    })
+                    .With(new Win32PlatformOptions
+                    {
+                        EnableMultitouch = true, 
+                        UseWgl = false, 
+                        AllowEglInitialization = false
+                    })
+                    .With(new VulkanOptions() { 
+                        UseDebug = true, 
+                        PreferDiscreteGpu = true 
+                    })
+                    .UseSkia()
```

Comment:
You could create a UseVulkan extension method that calls internally:
```csharp
                   return builder.UseSkia()
                    .With(new SkiaOptions()
                    {
                        CustomGpuFactory = VulkanSkiaGpu.CreateGpu
                    });
```

Reply:
I've added it.